### DISCUSSION TITLE
Does gino support mysql?

### DISCUSSION THREAD

**Discussion by zhenzi0322** [python-gino/gino#741] [Q&A] [answered]
Does gino support mysql?

**Comment by fantix** [comment, marked as answer]
Yes, it's in beta testing. Please try the [master branch](https://github.com/python-gino/gino) or [1.1.0-beta2](https://pypi.org/project/gino/1.1.0b2/). GINO 1.4/2.0 will inherit the MySQL support from upstream SQLAlchemy when it's ready (sqlalchemy/sqlalchemy#5747).
